src/gui/optionsdialog.ui

```diff
@@ -2665,7 +2665,7 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                   <string extracomment="minutes"> min</string>
                  </property>
                  <property name="maximum">
-                  <number>9999999</number>
+                  <number>5259487</number>
```

Comment:
Because of Qt spinboxes's stupid behavior which you press down 9 and it stops at 6 digits instead of showing the real max, i'd keep the previous number and adjust MAX_ETA to that *60. It's not like there is any benefit to having exactly 10 years limit right? Just a suggestion, no big deal really.
Other than this LGTM.

Reply:
Thank you for the input! Good point.

If you say we should not be afraid of it, I will change it! I was just kinda scared because of the previous 1-year limit. The int/long allows for large values but when I think of "large values", there is never a too large in programs.

Reply:
Alright, I changed it, it seems to work, let me know what you guys think. Thank you once again! I added some notes into the latest commit.